src/vision/status_quo/problems.md

```diff
@@ -0,0 +1,41 @@
+# Problems
+
+The following sections break out the biggest problems that we see.  These problems are grouped into categories, but they also show up in the various narratives.
+
+| Category | Problem |
```

Comment:
What about io-uring support? This came up in a conversation I had today and I realized we weren't really talking about it anymore.